alpha/lib/model/entry.php

```diff
@@ -149,7 +149,11 @@ class entry extends Baseentry implements ISyncableFile, IIndexable, IOwnable, IR
 	private $data_content = null;
 	
 	private $desired_version = null;
-	
+
+	//An attribute that contains the rules by which this entry was created, only in case it was created
+	// via clone operation
+	private $cloned_options = array();
```

Comment:
I would set it to null by default, in order to know if the entry cloned without any options.